registry/client.go

```diff
@@ -62,6 +65,31 @@ type credentials struct {
 	password string
 }
 
+type schemaInfoPayload struct {
+	Schema  string `json:"schema"`
+	ID      int    `json:"id"`
+	Version int    `json:"version"`
+	Subject string `json:"subject"`
+}
+
+func (s *schemaInfoPayload) Parse() (SchemaInfo, error) {
+	info := SchemaInfo{
+		ID: s.ID,
+		Version: s.Version,
+		Subject: s.Subject,
+	}
+	schema, err := avro.Parse(s.Schema)
+	info.Schema = schema
+	return info, err
+}
+
+type SchemaInfo struct {
+	Schema  avro.Schema
+	ID      int
+	Version int
+	Subject string
```

Comment:
Dont think we need this. I dont particularly like passing info back that was given to us.